tests/utils/test_xml_helpers.py

```
@@ -0,0 +1,65 @@
+import unittest
+import xml.etree.ElementTree as ET
+from datetime import datetime
+from src.utils.xml_helpers import create_opml_tree
+from src.models.feed import Feed
+
+class TestXMLHelpers(unittest.TestCase):
+
+    def setUp(self):
```

Ruff (0.8.0)

Missing return type annotation for public function `setUp`

Add return type annotation: `None`

(ANN201)